doc/modules/decomposition.rst

```diff
@@ -695,13 +695,22 @@ the mean of all elements of the data), and NNDSVDar (in which the zeros are set
 to random perturbations less than the mean of the data divided by 100) are
 recommended in the dense case.
 
-:class:`NMF` can also be initialized with random non-negative matrices, by
-passing an integer seed or a ``RandomState`` to :attr:`init`.
+:class:`NMF` can also be initialized with correctly scaled random non-negative
+matrices. An integer seed or a ``RandomState`` can also be passed to
+:attr:`random_state` to control reproducibility.
 
-In :class:`NMF`, sparseness can be enforced by setting the attribute
-:attr:`sparseness` to ``"data"`` or ``"components"``. Sparse components lead to
-localized features, and sparse data leads to a more efficient representation of
-the data.
+In :class:`NMF`, L1 and L2 priors can be added to the loss function in order
+to regularize the model. As in :class:`ElasticNet`, we control the combination
+of L1 and L2 with the :attr:`l1_ratio` parameter, and the intensity of the
+regularization with the :attr:`alpha` parameter. Then the priors terms are:
+
+.. math::
+    \alpha \rho ||W||_1 + \alpha \rho ||H||_1 + \frac{\alpha(1-\rho)}{2} ||W||_2 ^ 2
+    + \frac{\alpha(1-\rho)}{2} ||H||_2 ^ 2
```

Comment:
unclear that rho corresponds to `l1_ratio`. How would the eq. look with \text{l1_ratio}?